bin/ncm-ci.js

```diff
@@ -7,6 +7,10 @@ const graphql = require('../lib/graphql')
 const chalk = require('chalk')
 const fetch = require('node-fetch')
 
+// Setting up a timeout to be used with fetch() – will use the NCM_TIMEOUT environment
+// variable or default to 2 minutes (120 seconds / 120000 milliseconds)
+let NCM_TIMEOUT = process.env.NCM_TIMEOUT ? process.env.NCM_TIMEOUT : 120000
```

Comment:
```suggestion
const timeout = process.env.NCM_TIMEOUT
  ? Number(process.env.NCM_TIMEOUT) * 1000
  : 120000
```

- it should be `const` instead of a `let` because this variable's value won't change
- while bash variables usually follow `THIS_CASE`, their representation in javascript should follow `thatCase`
- as mentioned by @pmuellr, cast the value to a `Number` and assume it's written in seconds

Reply:
still need to check for a string in the env var:

```console
$ node -p 'isNaN(Number("foo"))'
true
```